controllers/nginx/pkg/config/config.go

```diff
@@ -301,6 +304,7 @@ func NewDefault() Configuration {
 		SSLSessionTimeout:        sslSessionTimeout,
 		UseGzip:                  true,
 		WorkerProcesses:          strconv.Itoa(runtime.NumCPU()),
+		LoadBalanceAlgorithm:     "least_conn;",
```

Comment:
I think it needs to be empty to specify the default, ngnix doesn't like leaving just a ; on that line